packages/medulas-react-components/src/components/PageLayout/index.tsx

```diff
@@ -22,6 +26,13 @@ const PageLayout = ({ id, children, title, primaryTitle }: Props): JSX.Element =
       paddingTop={2}
       height="100%"
     >
+      {showBackArrow && (
+        <Block marginLeft={-2}>
+          <Typography link>
+            <ArrowBackIcon fontSize="large" onClick={onBack} />
```

Comment:
I do believe for having the same appearance as the hamburger menu you need to wrap it using `IconButton`